torch/ao/sparsity/experimental/pruner/base_pruner.py

```diff
@@ -15,17 +15,17 @@
     nn.Conv2d
 }
 
-def _module_to_path(model, layer, prefix=''):
+def _module_to_fqn(model, layer, prefix=''):
```

Comment:
This function already exists in https://github.com/pytorch/pytorch/blob/master/torch/ao/sparsity/sparsifier/base_sparsifier.py#L17. Can you move these functions to sparsifier/utils.py and reuse them here?